### DISCUSSION TITLE
Unable to make a test request from UI - CORS blocking

### DISCUSSION THREAD

**Discussion by cristianabs** [scalar/scalar#1742] [Q&A] [answered]
Hello me again.. 

I am facing an issue with CORS trying to fetch the information from the server, the following images show what my code looks like currently, the console logs and the response from the server using Curl.

I tried using the server URL in the origin Option and 'true' as values. I get the same response in the console.

<img width="891" alt="1" src="https://github.com/scalar/scalar/assets/6587063/e316a69b-ac58-43a0-966c-71911c8dd48b">
<img width="1208" alt="2" src="https://github.com/scalar/scalar/assets/6587063/fbaac465-b13c-49c9-8d88-1d92f9fa4269">
<img width="1027" alt="3" src="https://github.com/scalar/scalar/assets/6587063/034d210c-5c31-43a5-afc0-5df057adf80c">

Do you happen to have any clue on this?

**Comment by marclave** [comment]
hi @cristianabs ! you would need to pass in the scalar proxy to handle these kinds of cors issues, you can run the proxy yourself or use       https://api.scalar.com/request-proxy

https://github.com/scalar/scalar?tab=readme-ov-file#configuration

let me know if this helps :)

**Reply by cristianabs** [reply]
Hi @marclave thanks for your reply, I configured the local proxy (https://github.com/scalar/scalar/tree/main/packages/api-client-proxy) and it is working, however, the body is not loaded in the response.

<img width="1204" alt="Screenshot 2024-05-17 at 09 22 22" src="https://github.com/scalar/scalar/assets/6587063/0885d56e-0cda-43ea-9066-325a522c9b6d">

is it something known?

**Reply by hanspagel** [reply, marked as answer]
Not know (until before your comment)! I’ve submitted a PR to resolve the issue:

https://github.com/scalar/scalar/pull/1797

Heads up, we’re about to switch to a new proxy. So you might want to switch to `proxy.scalar.com` soon, or host the Go proxy (read more in the linked PR) eventually: 

https://github.com/scalar/scalar/pull/1703
